styleguide/source/assets/scss/00-base/_forms.scss

```diff
@@ -39,6 +39,7 @@ textarea {
 
   &.error {
     border: 1px solid $red;
+    background-color: $white;
```

Comment:
This addresses the input value not being visible when there is an error on forms that have a dark background.